kola-1/WayFarer
# #167526504 implement LFA's feedback

#### What this PR does
Implement LFA's feedback and fix other issues

#### Tasks completed
- fix base url issue
- remove cors from dependencies
- fix npm vulnerability warning
- seperate testing database from development database
- remove unneccessary middlewares
- modify response data in busController.addBus

#### How this can be manually tested
- Clone the branch repo
- Change directory into it
- Run `npm install` to install all the dependencies
- Verify the cases above
- Run `npm run start-dev` to start the development server
- Test the cases above

[#167526504](https://www.pivotaltracker.com/story/show/167526504)
